src/azure-cli/azure/cli/command_modules/network/_help.py

```diff
@@ -12,6 +12,13 @@
 short-summary: Manage Azure Network resources.
 """
 
+helps['network list-service-tags'] = """
```

Comment:
One suggestion is that it will better to add a short summary for this command. Because if not, it will find short summary from SDK or other places and the fetched message use `Gets`, which is not consistent with [CLI command guideline](https://github.com/Azure/azure-cli/blob/dev/doc/authoring_help.md#tips-to-write-effective-help-for-your-command ) and other commands.

Reply:
now it will expose the SDK's help information to the user. Do you suggest us overwriting this help information?

Reply:
Right. Just my suggestion. It's not consistent with `list-usages` command when you run `az network -h` and don't follow CLI command guideline as you know.